src/main/java/io/logz/jmx2graphite/GraphiteClient.java

```diff
@@ -85,7 +89,13 @@ public static String sanitizeMetricName(String s, boolean keepDot) {
                 sb.append(c);
```

Comment:
I could add conditions to handle parentheses () and pipes |, however, it won't solve all cases.